packages/config/config/devices/0x0413/re_guard.json

```
@@ -0,0 +1,484 @@
+			"label": "Max Time (Present Mode)",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1440,
+			"defaultValue": 30
+		},
+		{
+			"#": "26",
+			"label": "Max Continous Time (Present Mode)",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32000,
+			"defaultValue": 30
+		},
+		{
+			"#": "27",
+			"label": "Macro Leakage Deactivation Time",
+			"description": "temporary macro leakage deactivation (e.g. filling pool)",
+			"unit": "h",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 168,
+			"defaultValue": 2
+		},
+		{
+			"#": "28",
+			"label": "Reaction to Macro Leakage",
+			"description": "0=disable detection, 1=blink LED, 2= blink LED and shut off valve",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable Detection",
+					"value": 0
+				},
+				{
+					"label": "Blink LED",
+					"value": 1
+				},
+				{
+					"label": "Blink LED and Shut Off Valve",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "29",
+			"label": "Flooding Sensor Leak Detected",
+			"description": "0-no leakage, 1- flooding sensor detected leakage",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		{
+			"#": "30",
+			"label": "Reaction to Flooding Sensor",
+			"description": "0=disable detection, 1=blink LED, 2= blink LED and shut off valve",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable Detection",
+					"value": 0
+				},
+				{
+					"label": "Blink LED",
+					"value": 1
+				},
+				{
+					"label": "Blink LED and Shut Off Valve",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "31",
+			"label": "Backup Battery Nominal Voltage",
+			"unit": "mV",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32000,
+			"defaultValue": 6000
+		},
+		{
+			"#": "32",
+			"label": "Backup Battery Nominal Capacity",
+			"unit": "mAh",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32000,
+			"defaultValue": 3000
+		},
+		{
+			"#": "37",
+			"label": "Time Set Correctly",
+			"description": "1 = time is set correctly, 0 = time is not set correctly",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0
+		},
+		{
+			"#": "38",
+			"label": "Total Operating Time",
+			"description": "Counting total operation time internally",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		{
+			"#": "39",
+			"label": "Operating Time Since Last Power On",
+			"unit": "s",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		{
+			"#": "49",
+			"label": "Auto Absent",
+			"description": "0 = disable, 1-168h time after no use, operation mode is changed to 1 (absent)",
```

```suggestion
			"label": "Auto Absent Timer",
			"description": "After no usage for this long, the operation mode is changed to 1 (absent)",
```
and 0 should be a single option ("Disable")